tests/test_cloudfront.py

```diff
@@ -311,3 +311,48 @@ def test_cloudfront_tagging_multi_region(self):
         west_resources = west_p.run()
 
         self.assertEqual(east_resources, west_resources)
+
+    def test_cloudfront_update_distribution(self):
+        factory = self.replay_flight_data("test_distribution_update_distribution")
+        p = self.load_policy(
+            {
+                "name": "cloudfront-tagging-us-east-1",
+                "resource": "distribution",
+                "filters": [
+                    {
+                        "type": "value",
+                        "key": "Logging.Enabled",
+                        "value": None,
+                    }
+                ],
+                "actions": [
+                    {
+                        "type": "update-distribution",
+                        "update": {
+                            "Comment": "",
+                            "Enabled": True,
+                            "Logging": {
+                                "Enabled": True,
+                                "IncludeCookies": False,
+                                "Bucket": 'test-enable-logging-c7n.s3.amazonaws.com',
+                                "Prefix": '',
+                            }
+                        }
+                    }
+                ],
+            },
+            config=dict(region='us-east-1'),
+            session_factory=factory,
+        )
+
+        resources = p.run()
+
+        self.assertEqual(len(resources), 1)
+
+        client = local_session(factory).client("cloudfront")
+        resp = client.list_distributions()
```

Comment:
instead of doing list_distributions again, just use the resources[0]['Id'] when fetching the distribution config

if there was any waiting involved (ie update didn't show immediately), use 
```
if self.recording:
     time.sleep(10)
```